src_c/_sdl2/video.pyx

```diff
@@ -764,3 +893,47 @@ cdef class Renderer:
                 raise error()
         else:
             raise error('target must be a Texture or None')
+
+    def blit(self, source, dest=None, area=None, special_flags = 0):
+        """ Only for compatibility.
+		Textures created by different Renderers cannot shared with each other!
+        :param source: A Texture or Surface.
+                        Surface will be converted by SDL_CreateTextureFromSurface.
+        :param area: Rect only, the portion of source texture.
+        :param special_flags: have no effect at this moment.
+        """
+        cdef SDL_Texture *tex
+        cdef Texture pyTex
+        if pgSurface_Check(source):
+            #Not recommended, creating texture every frame is expensive.
+            tex = SDL_CreateTextureFromSurface(self._renderer, 
```

Comment:
@husano896: If this has to be supported, maybe something like this?
```
__init__(self):   self.surfaceToTexture = {}


tex = self.surfaceToTexture.get(source)
if not tex:
    tex = SDL_CreateTextureFromSurface(self._renderer, pgSurface_AsSurface(source))
    self.surfaceToTexture[source] = tex
```

Reply:
@dlon Do you mean caching Surfaces as Textures?

if that case, should we limit the cache size (if len(self.surfaceToTexture) > MAX_CACHE_SIZE)?

Reply:
Yes, but maybe it should just raise an exception instead? Otherwise it either leaks memory or is very slow. I don't think it has to accept a Surface

Reply:
Maybe accepting surfaces is not a good idea, since Renderer is a new thing compared to SDL1.

My first thought was: if someone add Surfaces and Textures to a sprite Group, and uses Renderer as target, but not a problem if Surfaces are converted at first.